plugins/flytekit-vaex/setup.py

```diff
@@ -4,7 +4,11 @@
 
 microlib_name = f"flytekitplugins-{PLUGIN_NAME}"
 
-plugin_requires = ["flytekit>=1.3.0b2,<2.0.0", "vaex-core>=4.13.0,<4.14"]
+plugin_requires = [
+    "flytekit>=1.3.0b2,<2.0.0",
+    "vaex-core>=4.13.0,<4.14; python_version < '3.10'",
+    "vaex-core>=4.16.0,<4.17; python_version >= '3.10'",
```

Comment:
Are we able to use vaex [v4.17.1](https://pypi.org/project/vaex-core/4.17.1/#history)? Maybe <4.17 os too restrictive
```suggestion
    "vaex-core>=4.16.0; python_version >= '3.10'",
```

Reply:
I test on it. 4.17 is also okay for the python 3.10 installation. Just modify it.